kitsune/search/v2/views.py

```diff
@@ -0,0 +1,114 @@
+import json
+
+from django.conf import settings
+from django.http import HttpResponse
+from django.utils.translation import ugettext as _, pgettext
+
+from kitsune import search as constants
+from kitsune.search.forms import SimpleSearchForm
+from kitsune.sumo.api_utils import JSONRenderer
+from kitsune.sumo.utils import build_paged_url
+from kitsune.products.models import Product
+from kitsune.search.views import _fallback_results
+from kitsune.search.utils import locale_or_default
+from kitsune.search.v2.search import Search, KBConfig, AAQConfig
+
+
```

Comment:
pls add comments where it's not that straightforward why something it's the way it is